docs/core/examples/tls_alpn_server.py

```diff
@@ -56,7 +56,7 @@
 LISTEN_PORT = 8080
 
 
-class NPNPrinterProtocol(Protocol):
+class PrinterProtocol(Protocol):
```

Comment:
printer is kind of generic... but I don't have a better suggestion
```suggestion
class PrintNegotiatedProtocolProtocol(Protocol):
```